recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md

```diff
@@ -0,0 +1,70 @@
+    3. You understand all the metrics that matter for your product and you regularly ask insightful questions and find surprising answers. You care deeply about data and put significant effort into gathering it, cleaning it, querying it, and using it to make good decisions. You know _first_ when the numbers don't look good and you already have ideas on how to fix the root cause.
+    4. You leave your product in a solid state after launch. Debt is paid off, documentation cleaned up, and permanent owners identified/ramped up. Iterative improvements to your product continue. Your product does not wither and die. If it needs killing, you humanely decommission it and remove all traces so years later engineers aren’t tripping over the carcass.
+    5. You take full responsibility for everything related to your feature - every bug, outage, or unhappy customer. No excuses. 
+3. **_Customer Obsession: your effective partnership with PM levels-up our customer focus  _**
+    1. You know who your customers are and you evangelize their problems, needs, and vocabulary within your team. All members understand the bigger picture and the role of your project/team within it.
+    2. You understand the customer problems deeply enough to hold your own in a debate. You can predict what your customers will need and you use this to build technology that grows with them.
+    3. You are an effective partner to, and proxy for, your PM. You speak the same language and have the same vision. You complement each other naturally: s/he has breadth, you have depth; s/he can focus on long-term while you handle the short-term details. You help them find low-hanging product fruit through your technical expertise.
+    4. You know when to accept technical debt for the sake of the customer and you pay it down at the right moment. You honor organizational commitments made to the customer.
+    5. You escalate product requirements when they are unclear or not in the _overall _best interest of your customers/users.
+4. **_Leadership: sometimes engineering is more about people than about writing code…_**
+    1. You create positive pressure to ship. You help engineers see the _why_ so they put in maximum energy on the _what_ and the _how_. You have an intensity that is hard to describe and it energizes the people around you.
+    2. You know how to communicate perfectly: the message, the medium, the audience, the timing. Your stakeholders always know the project's status and tradeoffs chosen. Your partners/dependencies always know what you need from them and keep your requirements top-of-mind. Your leadership chain always knows the project's risks and how they can help accelerate you, if they so choose.
+    3. You keep the wheels greased. You have healthy relationships with all your partner teams, Tech Leads, dependencies, etc. Your project is never delayed because you get incorrectly blocked by external factors.
```

Comment:
" you get incorrectly blocked by external factors." - what does being correctly blocked mean? Maybe just remove "incorrectly"?

Reply:
I'll change it to 'unnecessarily'. Sometimes you get blocked and it's necessary, e.g. hardware doesn't show up on time b/c of a delay in the ports...